docs/source/tutorials/regression.ipynb

```diff
@@ -0,0 +1,676 @@
+    "Quickstart \n",
+    "\n",
+    "Cleanlab uses two inputs to generate scores for labels in the dataset:\n",
+    "- `labels`: NumPy array of given labels in the dataset. labels[i] should contain label for `i`-th example. \n",
+    "- `predictions`: NumPy array of predictions generated through your favorite regressor. predictions[i] should contain predicted value for `i`-th example. \n",
+    "\n",
+    "If you already have predictions from your regressor, you can generate label quality scores for each example using the code below: \n",
+    "\n",
+    "<div  class=markdown markdown=\"1\" style=\"background:white;margin:16px\">\n",
+    "\n",
+    "```python \n",
+    "\n",
+    "from cleanlab.regression.rank import get_label_quality_scores\n",
+    "label_quality_scores = get_label_quality_scores(labels, predictions)\n",
+    "\n",
+    "```\n",
+    "</div>\n",
+    "</div>"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Install dependencies and import them \n",
+    "You can use `pip` to install all the packages required for this tutorial as follows:\n",
+    "\n",
+    "`!pip install cleanlab xgboost`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# Package installation (hidden on docs website).\n",
+    "# Package versions we used: xgboost==1.7.2\n",
+    "\n",
+    "dependencies = [\"cleanlab\", \"xgboost\"]\n",
+    "\n",
+    "if \"google.colab\" in str(get_ipython()):  # Check if it's running in Google Colab\n",
+    "    %pip install cleanlab  # for colab\n",
+    "    cmd = \" \".join([dep for dep in dependencies if dep != \"cleanlab\"])\n",
+    "    %pip install $cmd\n",
+    "else:\n",
+    "    missing_dependencies = []\n",
+    "    for dependency in dependencies:\n",
+    "        try:\n",
+    "            __import__(dependency)\n",
+    "        except ImportError:\n",
+    "            missing_dependencies.append(dependency)\n",
+    "\n",
+    "    if len(missing_dependencies) > 0:\n",
+    "        print(\"Missing required dependencies:\")\n",
+    "        print(*missing_dependencies, sep=\", \")\n",
+    "        print(\"\\nPlease install them before running the rest of this notebook.\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "from xgboost import XGBRegressor"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# This cell is hidden on docs.cleanlab.ai\n",
+    "from sklearn.metrics import r2_score\n",
+    "from sklearn.model_selection import cross_val_predict\n",
+    "from cleanlab.regression.rank import get_label_quality_scores\n",
```

Comment:
do not include the imports in hidden cell, move to cell above

Reply:
corrected here: [79a509f](https://github.com/cleanlab/cleanlab/pull/572/commits/79a509f0d0bcdab86d84c77243d2abc61f38966a)